server/services/AlertService.js

```diff
@@ -98,7 +98,7 @@ export default class AlertService {
       return res.ok({
         body: {
           ok: false,
-          error: err.message,
```

Comment:
why are you renaming this to `resp`? `error` makes more sense

Reply:
Because I assumed the error message will be in `resp` filed in the helper function: https://github.com/opendistro-for-elasticsearch/alerting-kibana-plugin/blob/a8350e0f6842fcea59a41754c10a6648df392286/public/utils/helpers.js#L27, and the other error messages are all composed in `resp` field.
But I think what you thought make sense, I think I will change it back to `error`, and modified the helper function to not always access `resp` field as well.

Reply:
I also found some other names of `err` or `error` are changed by me in a previous PR, and I have changed them back.
Thank you for pointing it out.